test/unit/content-src/components/DiscoveryStreamComponents/DSCard.test.jsx

```diff
@@ -64,4 +64,8 @@ describe("<DSCard>", () => {
   it("should render DSLinkMenu", () => {
     assert.equal(wrapper.children().at(1).type(), DSLinkMenu);
   });
+
+  it("should start with no .active class", () => {
+    assert.equal(wrapper.find(".active").length, 0);
```

Comment:
Are the test changes intentional or accidental rebasing ?

Reply:
The test changes are intentional, just to bring up the coverage since the <FormattedMessage> component was removed. I'll probably open a new bug to write some better tests soon.